cfme/tests/openstack/infrastructure/test_api_port.py

```diff
@@ -11,5 +11,6 @@
 
 @pytest.mark.usefixtures("setup_provider_modscope")
 def test_api_port(provider, soft_assert):
-    soft_assert(provider.summary.properties.api_port.value.isdigit(),
+    port = provider.get_yaml_data()['port']
+    soft_assert(provider.summary.properties.api_port.value == port,
```

Comment:
**REQUIRED**
No use in having a soft_assert at the end of the test, since nothing else is dependent on it. I think it should be a full assert.